subt/tools/submit.py

```diff
@@ -19,6 +19,7 @@
     cp1 = "Cave Practice 1",
     cp2 = "Cave Practice 2",
     cp3 = "Cave Practice 3",
+    cs  = "Cave Circuit",
```

Comment:
why `cs`? `cc`, `cfinal` `cend`?

Reply:
I don't know... probably fatigue? Fixed in d619506